mne/preprocessing/maxwell.py

```diff
@@ -505,3 +536,97 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _read_fine_cal(info, fine_cal_fname):
+    """Read sensor locations and imbalances from fine calibration file."""
+
+    logger.info('Fine calibration file used in Maxwell Filter: ' +
+                fine_cal_fname)
+
+    # Read new sensor locations
+    with open(fine_cal_fname, 'r') as fid:
+        cal_chans = []
+        lines = fid.readlines()
+        lines = lines[::-1]
+        while len(lines) > 0:
+            line = lines.pop()
+            if line[0] != '#':
+                vals = np.fromstring(line, sep=' ')
+
+                # Check for correct number of items
+                if len(vals) not in [14, 16]:
+                    raise RuntimeError('Error reading fine calibration file')
+
+                ch = int(vals[0])
+                ch_name = 'MEG' + '%04d' % ch  # Zero-pad names to 4 characters
+                loc = vals[1:13]
+                imbalance = vals[13:]
+
+                cal_chans.append(dict(ch=ch, ch_name=ch_name, loc=loc,
+                                      imbalance=imbalance))
+
+    # Check that we ended up with correct number of channels
+    if len(cal_chans) != info['nchan']:
+        err_msg = '''Number of channels in fine calibration file (%i) does not
+            equal number of channels in info (%i)''' % (len(cal_chans),
+                                                        info['nchan'])
+        raise RuntimeError(err_msg)
+
+    return cal_chans
+
+
+def _update_sens_geometry(info, cal_chans):
+    """Helper to replace sensor geometry information"""
+    shift_list = []
+
+    # Replace sensor locations with those from fine calibration
+    for cal_chan in cal_chans:
+        ch_ind = info['ch_names'].index(cal_chan['ch_name'])
+        shift_list.append(info['chs'][ch_ind]['loc'] - cal_chan['loc'])
+        info['chs'][ch_ind]['loc'] = cal_chan['loc']
+
+    # Log quantification of sensor changes
+    shift_arr = np.array(shift_list)
+    logger.info('Mean change in sensor location and orientation: ' +
+                str(np.mean(shift_arr, 0)))
+    logger.info('Std. dev of change in sensor location and orientation: ' +
+                str(np.std(shift_arr, 0)))
+    logger.info('Max change in sensor location and orientation calculated as\n'
+                ' (np.max(np.abs(shift))): ' +
+                str(np.max(np.abs(shift_arr), 0)))
```

Comment:
I'd call it an "adjustment" instead of a change, it will make people more comfortable. I'm fine with getting those emails if they come, it's better that people know what's happening to their data in my mind.

@wronk how about this:
```
"Fine calibration adjusted coil positions by (mm): <mean> ± <std> SD (max: <max>)"
```
Why are there 14 values? I'd just operate on the flattened array.